### PR TITLE
[standard-ml/en-en] Format fixing and some APIs

### PR DESCRIPTION
**Pull request by neilsustc**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: standard-ml.html.markdown [hunk 1]
```diff
@@ -324,7 +329,11 @@ val n = op + (5, 5)   (* n is now 10 *)
 (* 'op' is useful when combined with high order functions because they expect
    functions and not operators as arguments. Most operators are really just
    infix functions. *)
-val sum_of_numbers = foldl op+ 0 [1,2,3,4,5]
+(* foldl f init [x1, x2, ..., xn]
+       returns
+       f(xn, ...f(x2, f(x1, init))...)
+       or init if the list is empty. *)
+val sum_of_numbers = foldl op + 0 [1, 2, 3, 4, 5]
```
Comment (vendethiel) [inline standard-ml.html.markdown]: `op +` reads like two different argument, that's confusing

File: standard-ml.html.markdown [hunk 2]
```diff
@@ -5,6 +5,7 @@ contributors:
     - ["David Pedersen", "http://lonelyproton.com/"]
     - ["James Baker", "http://www.jbaker.io/"]
     - ["Leo Zovic", "http://langnostic.inaimathi.ca/"]
+    - ["Yu Zhang", "http://neilsustc.github.io/"]
```
Comment (vendethiel) [inline standard-ml.html.markdown]: we usually reserve the contributor tag to people who've added significant content

**Comment by neilsustc** [comment]
Thanks for your review, I'll fix them tomorrow.

**Comment by neilsustc** [comment]
updated 👌

**Comment by vendethiel** [comment]
Amazing, thanks!

**Comment by vendethiel** [comment]
🍰

**Pull request merged**